docs/README.md

```diff
@@ -18,6 +18,28 @@ The development of this publicly-accessible client is being managed by Caltrans'
 
 The application is accessible to the public at [benefits.calitp.org](https://benefits.calitp.org).
 
+### Adoption by transit agencies
+
+The following California transit agencies have launched Cal-ITP Benefits for their riders, with the following enrollment pathway options:
+
+| Transit agency                                  | Older adults | Agency card | Veterans | Initial agency launch |
+| ----------------------------------------------- | ------------ | ----------- | -------- | --------------------- |
+| **Monterey-Salinas Transit**                    | Live         | Live        | Live     | 05/2021                |
+| **Santa Barbara Metropolitan Transit District** | Live         | In test     |          | 10/2023               |
+| **Sacramento Regional Transit District**        | In test      |             |          |                       |
```

Comment:
I'm wondering if we might want to track more than just the initial launch date, as in track the dates that a specific enrollment pathway was launched for a specific transit agency. So the table would be more like:
`| Transit agency | Enrollment pathway | Launch date |`

I think what you have here is a good start though, so I'll leave this as a thought that can be a future PR if we want

Reply:
Yea we could but that's so granular, and it kinda really only affects MST - since most agencies don't offer a Veterans discount. We're trying to phase out Agency Cards, and perhaps in the future, more agencies can go live with multiple enrollment pathways so we don't have to be doing these incremental rollouts.